package.json

```diff
@@ -897,7 +897,7 @@
                 "packageDownloadUrl": "https://repo1.maven.org/maven2",
                 "packageGroupId": "com.azure",
                 "packageArtifactId": "azure-ai-textanalytics",
-                "packageVersion": "1.0.0-beta.4",
```

Comment:
why did it jump from 1.0.0-beta.4. 
We had 5.0.0 GA release in between?
@scbedd

Reply:
I think that explain why TA in Microsoft.Doc site is teill 1.0.0-beta.4,
https://docs.microsoft.com/en-us/java/api/com.azure.ai.textanalytics.models?view=azure-java-preview
but need to be confirmed by Scott.
Hopefully, the Microsoft.Doc site has the latest version after this PR merged.

Reply:
There is this stable for TA - https://docs.microsoft.com/en-us/java/api/com.azure.ai.textanalytics.models?view=azure-java-stable
@mssfang - But is 1.0.1 the latest stable for TA?